### ISSUE TITLE
CantusDB's APIs should include column for syllabized full text

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#652]
Cantus Ultimus should have access to syllabized full texts where they exist, in order to properly display text-and-melody previews. This currently doesn't happen, as OldCantus doesn't include a column for exporting syllabized full texts in the CSV Export API. We should add this row, and also ensure that syllabized full texts can be accessed where it makes sense.

**Comment by jacobdgm** [comment]
Assigning a priority of "medium" because this is not urgent for the launching of NewCantus, but it's directly relevant to work going on on Cantus Ultimus.

Tagging @dchiller so he's aware of this issue being opened, and in case he has further comments.

**Comment by jacobdgm** [comment]
@dchiller, is this still necessary with the new syllabification library? Guessing yes - if this is the case, please unassign yourself and assign me.

**Comment by dchiller** [comment]
> @dchiller, is this still necessary with the new syllabification library? Guessing yes - if this is the case, please unassign yourself and assign me.

Yes, I think so: if someone has corrected a syllabification in CantusDB, we want that corrected syllabification in Cantus Ultimus. The most helpful thing here would be for it to be an additional column in the csv export, since that's what Cantus Ultimus currently uses, but also open to another solution if you think it would make sense.

I'll re-assign to you for now, then...

**Comment by kyrieb-ekat** [comment]
april25 
- add missing header
- add fix on CU end to expect new header and show the (newly) syllabified text
